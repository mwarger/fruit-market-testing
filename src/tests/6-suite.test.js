import { assert, describe, expect, it } from 'vitest';

describe('Tests about Foo', () => {
	it('foo', () => {
		assert.equal(Math.sqrt(4), 2);
	});

	it('bar', () => {
		expect(1 + 1).eq(2);
	});

	it('snapshot', () => {
		expect({ foo: 'bar' }).toMatchSnapshot();
	});
});
