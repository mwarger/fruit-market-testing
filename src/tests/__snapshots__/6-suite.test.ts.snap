// Vitest Snapshot v1

exports[`Tests about Foo > snapshot 1`] = `
{
  "foo": "bar",
}
`;
